_posts/2022-06-17-turing-elect.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Introducing TuringElect"
```

Comment:
What's the purpose of this post? Are we calling any help from the community towards the project or so?

Reply:
Nah, just that @yebai asked me to create a post providing information for the community about the project.